pkgs/development/python-modules/pyxlsb/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "pyxlsb";
+  version = "1.0.10";
+
```

Comment:
Per https://github.com/NixOS/nixpkgs/issues/124051, it's not clear what the desired default behavior will be, but there's a desire to add `format = "setuptools";` explicitly.